app/routes/me.tsx

```diff
@@ -11,6 +11,14 @@ import {
 } from '../utils/misc'
 import {getMagicLink, updateUser} from '../utils/prisma.server'
 import {requireUser, rootStorage, signOutSession} from '../utils/session.server'
+import {H2, H6} from '../components/typography'
+import {Grid} from '../components/grid'
+import {Input, Label} from '../components/form-elements'
+import {Button} from '../components/button'
+import {CheckIcon} from '../components/icons/check-icon'
+import {TEAM_MAP} from './signup'
```

Comment:
I want to avoid importing/exporting things from route modules. If we need to share, put it in `utils`. When we're all done, I'll come back around and organize things a little bit (hopefully by then we have [this](https://github.com/remix-run/remix/issues/10)).